src/features/unified-share-modal/flowTypes.js

```diff
@@ -264,6 +264,11 @@ type EmailFormTypes = {
     sendSharedLinkError: React.Node,
 };
 
+export type USMConfig = {
+    /** Whether the "Email Shared Link" button and form should be rendered in the USM/USF */
+    showEmailSharedLinkForm: boolean,
+};
+
```

Comment:
question: Do we expect more settings to get added in future to this `USMConfig` object?

Reply:
Good question @manishPh. It's definitely a possibility, so I wanted to make the configuration object extensible.